src/index.js

```diff
@@ -24,8 +24,16 @@ class Form {
         destroy.call(this);
     }
     
+    /**
+     * @method getFormData
```

Comment:
@SimplySayHi let me know what should I put here as methods description and return value